admin/controllers/content/views/order/view.php

```diff
@@ -0,0 +1,151 @@
+                                // some fields will currently parse json config files to determine tables to query for friendly values
+                                // PER row/field. not ideal. ESPECIALLY IMPORTANT FOR ORDER VIEW SINCE ALL CONTENT ITEMS SHOWN
+                                echo $curfield->get_friendly_value($named_custom_fields[$propname]); // pass named field custom field config to help determine friendly value
+                            }
+                            else {
+                                echo $i->{$content_list_field->name}; // field contents from DB
+                            }
+                            ?>
+                        </td>
+                    <?php endforeach; ?>
+                <?php endif; ?>
+            </tr>
+        <?php endforeach; ?>
+    </tbody>
+</table>
+
+<script src="https://cdn.jsdelivr.net/npm/sortablejs@1.15.6/Sortable.min.js"></script>
+<script>
+    new Sortable(document.getElementById('ordertablebody'), {
+        animation: 150,
+        onUpdate: function (e) {
+            /* console.log("From ", e.oldIndex, " to ", e.newIndex);
+            console.log(e.item.dataset); */
+            // update all ordering indices to ensure correctness
+            // get arr of ids
+            let id_arr=[];
+            let content_type=<?=$content_type;?>;
+            document.querySelectorAll('.orderitem').forEach((e)=>{
+                id_arr.push(e.dataset.content_id);
+            });
+            let id_arr_string = id_arr.join(",");
+            if (id_arr_string) {
+                // pass to server
+                console.log(id_arr_string);
+                
+                async function update_ordering() {
+                    const formData = new FormData();
+
+                    // orderall
+                    // can be used to pass ALL ids in content for ordering
+					//formData.append("ids", id_arr_string);
+					//formData.append("action", "setorderall");
+					//formData.append("content_type", <?=$content_type;?>);
+
+                    // changeorder
+                    // id, new_order, prev_order, content_type
+                    formData.append("action", "changeorder");
+                    formData.append("id", e.item.dataset.content_id);
+                    formData.append("content_type", <?=$content_type;?>);
+                    formData.append("new_order", e.newIndex+1); // client-size index 0 based, ordering 1
+                    formData.append("prev_order", e.oldIndex+1);
+                    fetch("<?=Config::uripath();?>/admin/content/api", {
+                            method: "POST",
+                            body: formData,
+                    }).then((response) => response.json()).then((data) => {
+                        console.log(data);
+                    });
+                }
+
+                update_ordering();
+            }
+        },
+    });
+</script>
```

Comment:
put this in its own js file, use window.uripath

Reply:
happy for content_type to be added to window object and used this way as well?